firmware/network/net_tools/net_tools.c

```diff
@@ -99,7 +102,8 @@ int8_t set_ipv6_multicast(kernel_pid_t iface_index, ipv6_addr_t ip, uint8_t pref
                "File: %s, Function: %s, Line: %d\n",
                __FILE__, __func__, __LINE__);
     }
-    uint16_t flags = GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_VALID | (prefix << 8U);
+    uint16_t flags = GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_VALID |
+(prefix << CONFIG_PREFIX_DISPLACEMENT);
```

Comment:
I tested this with border_router test, and the change of prefix_displacement throws an error. Please bring some details about this implementation